Client/Frontend/Browser/BrowserViewController/BVC+Rewards.swift

```diff
@@ -292,28 +292,26 @@ extension Tab {
         
         let group = DispatchGroup()
         group.enter()
-        
-        webView.evaluateSafeJavaScript(functionName: "document.documentElement.outerHTML.toString", contentWorld: .defaultClient) { html, _ in
+
+        webView.evaluateSafeJavaScript(functionName: "new XMLSerializer().serializeToString(document)", contentWorld: .defaultClient) { html, _ in
             htmlBlob = html as? String
             group.leave()
         }
-        
+
         if shouldClassifyLoadsForAds {
             group.enter()
-            webView.evaluateSafeJavaScript(functionName: "document.body.innerText", contentWorld: .defaultClient, asFunction: false) { text, _ in
-                // Get the list of words in the page and join them together with a space
-                // to send to the classifier
-                classifierText = (text as? String)?.words.joined(separator: " ")
+            webView.evaluateSafeJavaScript(functionName: "document?.body?.innerText", contentWorld: .defaultClient, asFunction: false) { text, _ in
```

Comment:
Updated to match desktop

Reply:
Are you sure about this change? `document.body` shouldn't ever really be null, so not sure if you require the optional chaining operator (`?`) here.